pkg/workload/workloadsql/workloadsql.go

```diff
@@ -36,6 +36,18 @@ import (
 func Setup(
 	ctx context.Context, db *gosql.DB, gen workload.Generator, l workload.InitialDataLoader,
 ) (int64, error) {
+	var hooks workload.Hooks
+	if h, ok := gen.(workload.Hookser); ok {
+		hooks = h.Hooks()
+	}
+	// Added for multi-region TPC-C. Used to convert the database to a multi-
```

Comment:
nit: eh i kind of think this comment doesn't need to be here, since later down the track it may be used for non-multi-region-TPC-C things